content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -147,9 +147,12 @@ For more information on version skews, see:
 
 {{< tabs name="k8s_install" >}}
 {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+
+Commands must be run as [superuser](http://manpages.ubuntu.com/manpages/xenial/man8/sudo_root.8.html).
```

Comment:
i'd update this like so, to be:
- distro agnostic
- to have the sentence in a "note block"

```
{{< note >}}
All commands must be run as a super user.
{{< /note >}}
```